internal/ingress/controller/store/store_test.go

```diff
@@ -972,7 +970,7 @@ func TestListIngresses(t *testing.T) {
 			ObjectMeta: metav1.ObjectMeta{
 				Name:              "test-3",
 				Namespace:         "testns",
-				CreationTimestamp: sameTime,
+				CreationTimestamp: metav1.NewTime(time.Now()),
```

Comment:
@alexkursell this is millisecond granularity right? Can they not potentially end up being equal still?